lib/metasploit/framework/login_scanner/redis.rb

```diff
@@ -15,12 +15,48 @@ class Redis
         include Metasploit::Framework::LoginScanner::RexSocket
         include Metasploit::Framework::Tcp::Client
 
+        attr_writer :older_redis
+
+        # Returns whether the scanner is working with an older redis version or not.
+        # @return [Boolean] True if using an older Redis; otherwise false.
+        def older_redis
+          @older_redis || false
+        end
```

Comment:
I don't think we need to make this part of the public API - seems like a leaky implementation detail 👀

Reply:
hidden into just using `@older_redis` and keeping the implementation internal.